routes/trading.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express')
+const router = express.Router()
+const authenticate = require('../middlewares/authenticate')
+const { trade } = require('../controllers/tradingController')
+const { newTrade } = require('../middlewares/validators/trading')
+
+/**
+ * @swagger
+ * /trade:
+ *  patch:
+ *   summary: Used for new trading request
+ *   tags:
+ *     - Trading
```

Comment:
We need to write this as we are using cookies, right?
` security:
       - bearerAuth: []`